src/model/user-course.model.ts

```diff
@@ -0,0 +1,49 @@
+import {
+  Entity,
+  Column,
+  PrimaryGeneratedColumn,
+  CreateDateColumn,
+  UpdateDateColumn,
+  DeleteDateColumn,
+} from 'typeorm'
+
+// import UserModel from './users.model'
+
+@Entity('user-course')
+export default class UserCourseModel {
+  @PrimaryGeneratedColumn()
+  id: number
+
+  @CreateDateColumn({name: "created_at"})
+  createdAt: Date
+
+  @UpdateDateColumn({name: "updated_at"})
+  updatedAt: Date
+
+  @DeleteDateColumn({name: "deleted_at"})
+  deletedAt?: Date
+
+  // // Foreign key
+  // @ManyToOne(() => UserModel)
+  // @JoinColumn({name: "user_id"})
+  // _user: UserModel
```

Comment:
If we're not using this we should probably delete it.

Reply:
Updated in commit